### ISSUE TITLE
Endless range mutation cycles

### ISSUE THREAD

**Issue by dgollahon** [mbj/mutant#1035]
Since endless ranges no longer crash mutant (as reported in #1026), I have tried switching several parts of my codebase to use endless ranges. Unfortunately, mutant mutates `some_array[1..]` to `some_array[1...]`. If I switch to the latter it mutates it back in the reverse direction. If you run the example from #1026, you will see this occur. It is my understanding that `1..` and `1...` generally have the same semantics, though they are not equal ([see the docs](https://ruby-doc.org/core-2.6/Range.html#class-Range-label-Endless+Ranges)). This effectively prevents me from using endless ranges with mutants because I can't see any reasonable way to kill these.

I think endless ranges should not be mutated from inclusive to exclusive at all or, if they must, it should be unidirectional so that the mutation can be killed.

**Comment by mbj** [comment]
To be fixed in #1027.

**Comment by mbj** [comment]
This was fixed in#1036 and released as 0.9.11.

**Comment by mbj** [comment]
OT: in theory there is an observable difference:

```
irb(main):005:0> (1..).exclude_end?
=> false
irb(main):006:0> (1...).exclude_end?
=> true
```

But thats not really something to assert from specs. Hence inclusive vs exclusive on endless range is a style issue, outside of mutants semantic scope.

So its correct to remove the mutation.
